platform/android/app/src/main/java/coop/polypoly/polypod/PodUnlockActivity.kt

```diff
@@ -21,7 +21,7 @@ class PodUnlockActivity : AppCompatActivity() {
     }
 
     private fun authenticate() {
-        Authentication.authenticate(this, newStatus = false) {
+        Authentication.authenticate(this, showAuthTexts = true) {
```

Comment:
This has changed, too. While in the other places it seems to be a simple refactoring, here it seems to show a different semantics.

Reply:
yes, as described above, we want different paths to show different UI texts to reflect the flow diagram.